packages/components/addon-main.cjs

```diff
@@ -10,6 +10,7 @@ module.exports = {
   ...addonV1Shim(__dirname),
   contentFor(type, config) {
     if (
+      !config.flightIconSpriteLazyEmbed &&
       !config.emberFlightIcons?.lazyEmbed &&
```

Comment:
[suggestion] should we add a comment here about the two similarly named config parameters, and the fact that one is the new (correct) one and the other is the old and deprecated one?

Reply:
[question] also, would it male sense to define a `const legacyLazyEmbed` like below? (the variable's name makes it clear that it's a legacy thing)